config.default

```diff
@@ -132,6 +132,15 @@ mpasInterpolationMethod = bilinear
 startYear = 1
 endYear = 9999
 
+[index]
+## options related to producing nino index.
+
+# start and end years for the nino 3.4 analysis.  Using out-of-bounds values
+#   like start_year = 1 and end_year = 9999 will be clipped to the valid range
+#   of years, and is a good way of insuring that all values are used.
+startYear = 2
+endYear = 6
```

Comment:
Is there a reason to retain so many config files for each system?  I have updated the newest ones in each directory.  do I need to do all?  If so, I'd like to understand the need for all of them (for example configs/edison has 4 config files)